docs/account_management/cheaha_account.md

```diff
@@ -39,7 +39,7 @@ If your account is in good standing you should see a page like the following.
 
 ### Account Requires Certification
 
-We review accounts periodically to ensure the system is being used fairly and as intended. Part of this process is to certify that researchers with accounts still wish to make use of Cheaha. Once per year every researcher will be required to certify their account before making use of Cheaha. If your account requires certification, when logging in to our [Open OnDemand](../cheaha/open_ondemand/index.md) web portal you will see the following notification page.
+We review accounts periodically to ensure the system is being used fairly and as intended. Part of this process is to certify that researchers with accounts still wish to make use of Cheaha. Once per year every researcher will be required to certify their account before making use of Cheaha. In addition, an account may also require certification after it has been placed on hold (see our [Account on Hold section](#account-on-hold)). This ensures that you, the user, are engaged in the process of reactivating your account to use our resources. If your account requires certification, when logging in to our [Open OnDemand](../cheaha/open_ondemand/index.md) web portal you will see the following notification page.
```

Comment:
Good start! Please remove "may also" from the first sentence. This applies to all accounts so "an account" should be "accounts". Then rephrase that sentence appropriately for consistency.

The second sentence is wordy and may be unnecessary. It also helps to talk in terms of benefits. By requiring a human push a button, we're improving security by making it much harder to automate the reactivation process. Can you find a very concise way to phrase that?